tests/test_sota_checkpoints.py

```diff
@@ -23,6 +24,9 @@
 DIFF_FP32_MAX_GLOBAL = 0.1
 
 METRICS_DUMP_PATH = PROJECT_ROOT / 'metrics_dump'
+OPENVINO_DIR = os.path.join(os.path.dirname(PROJECT_ROOT), 'intel', 'openvino')
+ACC_CHECK_DIR = os.path.join(OPENVINO_DIR, 'deployment_tools', 'open_model_zoo', 'tools', 'accuracy_checker')
```

Comment:
Here, and from now on everywhere, use pathlib instead of os.path. The `METRICS_DUMP_PATH` is an example of such usage.

Reply:
Done